src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/tasks/CleanupEnvelopesDlqTaskTest.java

```diff
@@ -120,4 +125,26 @@ public void should_not_process_messages_when_connection_exception_is_thrown() {
         assertThat(exception).isNull();
     }
 
+    @Test
+    public void should_log_messageId_for_invalid_messages() throws Exception {
+        //given
+        given(message.getEnqueuedTimeUtc())
+            .willReturn(LocalDateTime.now().minus(ttl.plusSeconds(10)).toInstant(ZoneOffset.UTC));
+        String messageId = UUID.randomUUID().toString();
+        given(message.getMessageId()).willReturn(messageId);
+        given(message.getBody()).willReturn("```".getBytes()); //invalid message
+        given(messageReceiver.receive()).willReturn(message).willReturn(null);
+
+        //when
+        cleanupDlqTask.deleteMessagesInEnvelopesDlq();
+
+        //then
+        assertThat(capture.toString())
+            .containsPattern(
+                ".+ ERROR \\[.*\\].*"
+                    + CleanupEnvelopesDlqTask.class.getSimpleName()
+                    + " An error occurred while parsing the dlq message with Message Id: " + messageId
+            );
+    }
```

Comment:
let's hope this test works as logging library is v4 😳

Reply:
@doncem removed the test